mkdirp/test.ts

```diff
@@ -0,0 +1,30 @@
+import { cwd, lstat, makeTempDirSync, removeAll, FileInfo } from "deno";
+import { test, assert } from "https://deno.land/x/testing/testing.ts";
+import { mkdirp } from "./mkdirp.ts";
+
+let root: string = `${cwd()}/${Date.now()}`; //makeTempDirSync();
```

Comment:
I guess there's some issue with makeTempDirSync?

Reply:
when i was using `makeTempDirSync` the mac test job exited with 1 and reported that "/var is not a directory". is this workaround okay for the meantime?

Reply:
Yep! Just landed now.

Reply:
Thanks for your patience!